index.d.ts

```diff
@@ -10,7 +10,7 @@ declare namespace sortKeys {
 		/**
 		[Compare function.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)
 		*/
-		readonly compare?: (left: string, right: string) => number;
+		readonly compare?: (left: string, right: string, context?: object) => number;
```

Comment:
```suggestion
		readonly compare?: (left: string, right: string, context?: unknown) => number;
```

Since we cannot know the type.

Reply:
The `context` parameter should be documented. See: https://github.com/sindresorhus/typescript-definition-style-guide